README.md

```diff
@@ -11,6 +11,41 @@ yarn add react-state-helpers
 
 ## Higher Order Component Wrapper
 
+You can add `react-state-helpers` to any project quickly and easily with the supplied decorator.
+
+```js
+import React, { Component } from 'react';
+import wrapStateHelpers from 'react-state-helpers';
+
+// login could be a function that takes on object with the keys:
+// userName, and password
+import { login } from 'src/api';
+
+@wrapStateHelpers
+export default class Example extends Component {
+  render() {
+    const {
+      handleSubmit, mut,
+      values: { userName }
+    } = this.props;
+
+    return (
+      <div>
+        Welcome, { userName }!
+        <form onSubmit={handleSubmit(login)}>
+          <input name='userName' type='text' onChange={mut('userName')}/>
+          <input name='password' type='password' />
+
+          <input type='submit' value='Login' />
+        </form>
+      </div>
+    )
+  }
+}
+```
+
+If you are on an older version of Javascript you can use the following syntax for the same results...
+
```

Comment:
since these examples are identical, the following example should probably be truncated, yeah?